pkg/ddevapp/config.go

```diff
@@ -1113,15 +1113,6 @@ ENV N_PREFIX=/home/$username/.n
 ENV N_INSTALL_VERSION="%s"
 `, app.NodeJSVersion)
 	}
-	// TODO: I would rather put this in the ddev-webserver image but...
-	// - We don't know the username yet there, so would have to put in /usr/local/composer
-	// - /usr/local/composer would then have to be set as COMPOSER_HOME
-	// - We could set the permissions here so people could use it
-	// - BUT... we don't know the PHP version until runtime.
-	// - Oh, but we don't know the PHP version here yet because it's not runtime yet.
-	// - Maybe this can be done or checked by `ddev xhgui` ?
-	// Add php-profiler for xhgui
-	extraWebContent = extraWebContent + "\nUSER ${username}\nRUN composer global require perftools/php-profiler --dev\nUSER root\n"
```

Comment:
Do we need to chown/chmod the installed php-profiler stuff in start.sh now that it's installed at image build time?

Reply:
I pushed the image (without running tests):

```
docker pull ddev/ddev-webserver:20250331_stasadev_xhgui_php-profiler
docker run -it --rm ddev/ddev-webserver-prod:20250331_stasadev_xhgui_php-profiler ls -la /usr/local/xhgui.collector
```